cypress/support/pages/teams/pageCommonTeams.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+class Teams_Common {
+
+  static #teamTitle = '[data-testid="title_of_an_element"]'
+  static #teamSettings = '[data-testid="team_settings"]'
+  static #manageTeamMembers = '[data-testid="administrate_team_members"]'
+  static #editTeam = '[data-testid="edit_team_members"]' //'i.fa-edit'
```

Comment:
Remove extra selectors in comments

Reply:
we may need to use them because of time to time elements with (data-test id) is hidden, but I removed it as comment now.